themes/schema.json

```diff
@@ -2726,6 +2727,31 @@
               }
             }
           }
+        },
+        {
+          "if": {
+            "properties": {
+              "type": {
+                "const": "gcp"
+              }
+            }
+          },
+          "then": {
+            "title": "GCP Segment",
+            "description": "https://ohmyposh.dev/docs/segments/gcp",
+            "properties": {
+              "properties": {
+                "properties": {
+                  "display_error": {
```

Comment:
Can be removed if the above comment is addressed.

Reply:
Section removed